### PR TITLE
Add recipe for spaghetti all'amatriciana

### PR DESCRIPTION
**Pull request by Jebusthebus**

### CODE REVIEW

**Comment by hallzy** [comment]
@Jebusthebus If you are wondering why the checks were failing, I think it is because the file is in DOS format.

If you have the tool dos2unix it will convert files to unix line endings, or I believe git has eol configs?

https://docs.github.com/en/github/using-git/configuring-git-to-handle-line-endings

The check script should make that clearer though. And maybe a git attributes file would be good for this repo to make sure that that is consistent.

File: src/spaghetti-all'amatriciana.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Spaghetti all'amatriciana
+
+Ignore the fact that those in the photo are pennette and not spaghetti, that's all i had that day.
```
Comment (hallzy) [inline src/spaghetti-all'amatriciana.md:3]:
```suggestion
Ignore the fact that those in the photo are pennette and not spaghetti, that's all I had that day.
```

**Comment by Jebusthebus** [comment]
@hallzy Thank you, I hoped my markdown editor would fix it but I guess I was mistaken.

File: src/spaghetti-all'amatriciana.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+- Guanciale preferably or Bacon 3.5oz (100g)
+- White wine 1/4 cup (60g)
+- Tomato passata 10oz (300g)
+- Spaghetti or similar long pasta 12oz (350g)
+- (optional) Extra pecorino romano or parmesan 
+
+## Directions
+
+1. Grate the pecorino and keep it aside.
+2. Mince the chili pepper/s and keep it/them aside.
```
Comment (hallzy) [inline src/spaghetti-all'amatriciana.md:25]:
```suggestion
2. Mince the chili pepper(s) and keep it/them aside.
```

File: src/spaghetti-all'amatriciana.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+3. Cut the guanciale in thin stripes and cook them in the oil with the minced pepper/s for about 3 minutes on medium/high heat.
```
Comment (hallzy) [inline src/spaghetti-all'amatriciana.md:26]:
```suggestion
3. Cut the guanciale in thin stripes and cook them in the oil with the minced pepper(s) for about 3 minutes on medium/high heat.
```

**Review by hallzy — CHANGES_REQUESTED** [review]

**Pull request merged**